wallettool/build.gradle

```diff
@@ -58,9 +61,7 @@ asciidoctor {
     }
 }
 
-def minGraalVMGradleVersion = GradleVersion.version("7.3")
```

Comment:
Why did this version change? Also the other version check changed too – why? A small comment in the commit description could clear this up.

Reply:
I originally used "7.3" because I was thinking to just keep things simple and use a very recent version. I later decided to switch to "6.8" because that's the actual minimum version for the feature used.

Reply:
>Also the other version check changed too – why? 

The `annotationProcessorMinVersion` check was incorrect. It was excluding "4.6" because of '> 0' so I switched it to `>= 0`.

Reply:
I added a commit with the fix right away. I think fixes should stay separate, this allows them to be merged quicker and also makes this PR simpler to review.